addons/settings/fnc_addSetting.sqf

```diff
@@ -45,4 +45,8 @@ Author:
     commy2
 ---------------------------------------------------------------------------- */
 
-call (uiNamespace getVariable QFUNC(init)) == 0
+if (isNil QFUNC(init)) then {
```

Comment:
Comment why?

Reply:
If this one function undefined (implies all others are too) then define all functions.
It's basically an include guard.

Reply:
Not comment for me (but probably also for me in a few months).